docs/Homebrew-brew-Maintainer-Guide.md

```diff
@@ -0,0 +1,102 @@
+# Homebrew/brew Maintainer Guide
+
+This document describes a few components of the `Homebrew/brew` repository that are useful for maintainers to
+be aware of, but don't necessarily need to appear in documentation for most users and contributors.
+
+## Reviewing PRs
+
+Using `gh pr checkout NUMBER` is a super easy way to check out a PR branch using the GitHub CLI.
+
+When reviewing a PR, use "comment", "approve", or "request changes" when submitting based on the following guidelines:
+
+- Comment: if the PR isn't quite ready to be merged
+- Approve: if you feel that the PR is in a good state to be merged, even if there are
+  non-blocking changes you'd like to be made
+- Request changes: if you feel strongly that the PR is likely to cause a problem for users or
+  have a genuine reason to oppose the PR.
+
+## Merging PRs
+
+Merging should be done using the standard Merge button in the `Homebrew/brew` repository to preserve history and GPG commit signing. The Squash and Merge and Rebase and Merge buttons are disabled.
+
+PRs must meet the following conditions to be merged:
+
+- Have at least one maintainer (or `BrewTestBot`) approval. See [below](#automatic-approvals)
+  for more details about how `BrewTestBot` approves PRs.
+- Have passing CI. This is a _mandatory_ step. PRs with failing CI should _never_ be merged.
+  See [below](#ci) for more information about `Homebrew/brew` CI.
+
+If possible, PRs should also have:
+
+- Linear commit history (i.e. no merge commits in PR branches).
+- GPG-signed commits (see the private `ops` repository for instructions on setting this up).
+
+## Automatic approvals
+
+To ensure that each PR has the opportunity to be seen and reviewed by any other maintainers who wish
+to take a look, all PRs require an approval before they can be merged. However, not every PR is
+reviewed by another maintainer, and some PRs are urgent enough that they need to be merged without
+an approval by another maintainer.
+
+As a compromise between always needing a review and allowing maintainers to merge PRs they deem ready,
+the `Triage` CI job will ensure that PRs cannot be merged until they've been open for 24 hours
+(only counting hours that occur during the business week). After the triage period has expired, the
```

Comment:
```suggestion
(only counting hours that occur during Monday to Friday). After the triage period has expired, the
```

Reply:
is "during" necessary here? Perhaps, `occur Monday to Friday`

Reply:
@maxim-belkin Agreed!